budgetr-backend/src/main/java/pl/mantiscrab/budgetr/domain/user/SignedInUserGetterImpl.java

```diff
@@ -0,0 +1,23 @@
+package pl.mantiscrab.budgetr.domain.user;
+
+import lombok.AllArgsConstructor;
+import pl.mantiscrab.budgetr.domain.user.exceptions.BudgetrUserNotAuthenticatedException;
+import pl.mantiscrab.budgetr.domain.user.infrastructure.SignedInUsernameGetter;
+import pl.mantiscrab.budgetr.domain.user.infrastructure.UserNotAuthenticatedException;
+
+@AllArgsConstructor
+class SignedInUserGetterImpl implements SignedInUserGetter {
+    private final SignedInUsernameGetter usernameGetter;
```

Comment:
This looks really confusing. Please consider renaming it

Reply:
What would you suggest?

Reply:
Just UserName